src/components/SequenceInput/SequenceInput.js

```diff
@@ -98,140 +130,78 @@ export default function SequenceInput({ onSaveClick }) {
-                !hasCorrectLowerThirdLength) ? (
-                <InfoTimeCode hasError>
-                  Timecode fehlt, ist fehlerhaft oder ist insgesamt zu lang!
-                </InfoTimeCode>
-              ) : (
-                <InfoTimeCode>&nbsp;</InfoTimeCode>
-              )}
+              <ValidationError
+                errorMessage="Timecode fehlt, ist fehlerhaft oder ist insgesamt zu lang!"
+                hasError={validateLowerThirdIn()}
+              />
               <TimecodeInput
                 style={{ margin: '10px 0' }}
                 disabled={disabled}
-                handleBackSpace={handleBackSpace}
+                onBackSpace={preventCursorJumpToEnd}
                 title="Bauchbinde Länge"
                 placeholder="SS:FF"
                 inputValue={formatter(timeCodeLowerThirdLength)}
                 onChange={(event) =>
                   handleTimeCodeChange(event, setTimeCodeLowerThirdLength)
                 }
               />
-              {isDirty &&
-              (!timeCodeLowerThirdLength ||
-                lowerThirdOutHasOnlyZeros ||
-                !isCorrectLowerThirdLength ||
-                !hasCorrectLowerThirdLength) ? (
-                <InfoTimeCode hasError>
-                  Timecode fehlt, ist fehlerhaft oder ist insgesamt zu lang!
-                </InfoTimeCode>
-              ) : (
-                <InfoTimeCode>&nbsp;</InfoTimeCode>
-              )}
+              <ValidationError
+                errorMessage="Timecode fehlt, ist fehlerhaft oder ist insgesamt zu lang!"
+                hasError={validateLowerThirdLength()}
+              />
             </LowerThirdContainer>
           </>
         )}
         <Actions>
-          <Delete onClick={onDeleteClick} type="reset">
+          <Delete onClick={onCancelClick} type="reset">
             ABBRECHEN
           </Delete>
-          <Save>SPEICHERN</Save>
+          {updateCard ? <Save>AKTUALISIEREN</Save> : <Save>SPEICHERN</Save>}
         </Actions>
       </form>
     </Wrapper>
   )
 
-  function handleBackSpace(event) {
-    if (event.key === 'Backspace' || event.key === 'Process') {
-      if (
-        /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(
-          navigator.userAgent
-        )
-      ) {
-        const caret = event.target.selectionStart
-        const element = event.target
-        window.requestAnimationFrame(() => {
-          element.selectionStart = caret
-          element.selectionEnd = caret
-        })
-      } else {
-        const caret = event.target.selectionStart
-        const element = event.target
-        window.requestAnimationFrame(() => {
-          element.selectionStart = caret - 1
-          element.selectionEnd = caret - 1
-        })
-      }
-    }
-  }
-
   function handleTimeCodeChange(event, timeCodeSetterFunc) {
     const { value } = event.target
-
     const formattedTimecCode = getTimeCodeUnFormatted(value)
```

Comment:
Might be a typo: Do you mean formattedTimeCode?

Reply:
Unbelievable that you have seen this. it took me a minute to see the typo 😂